Packages/io.chainsafe.web3-unity.web3auth/Runtime/WalletGUI/Scripts/Web3AuthWalletGUITxManager.cs

```diff
@@ -78,15 +79,20 @@ private void OnTransactionRequested(TransactionRequested transactionRequested)
         
         PromptTransactionRequest();
     }
-
-    private void PromptTransactionRequest()
+    
+    /// <summary>
+    /// Prompts transaction request display.
+    /// </summary>
+    private async void PromptTransactionRequest()
     {
         TransactionRequested transactionRequested = _transactionQueue.Peek();
         
         incomingTxNotification.SetActive(true);
 
         if (AutoConfirmTransactions)
         {
+            await Task.Delay(1000);
```

Comment:
Was there a more elegant way for this? :D Like an event or something?

Reply:
rob found it dw